### DISCUSSION TITLE
Dynamically building a form from a rest query

### DISCUSSION THREAD

**Discussion by adussarps** [Budibase/budibase#10121] [Help] [answered]
Hello there, I'm kinda new to BudiBase, but I'm trying to use it as a front end for a **"form application"**. 

To do this, I'm trying to have a page that will be able to display a **"dynamic form"** based on the result of a rest query. 

The query can be called when loading the page and answer a different result depending on some route parameters.

Is this something that can be doable with BudiBase?

(and if yes, how should we format the result of the rest query to be interpreted as a schema?)

Thanks in advance :)!

**Comment by melohagan** [comment, marked as answer]
Hey @adussarps 

This functionality would be great, but unfortunately it is not yet possible: https://github.com/Budibase/budibase/issues/5658
